app/services/ExactTargetService.scala

```diff
@@ -19,27 +20,28 @@ import scala.concurrent.duration._
 
 trait ExactTargetService extends LazyLogging {
   lazy val etClient: ETClient = ETClient
-  def zuoraService: ZuoraService
+
+  def subscriptionService: SubscriptionService
+  def paymentService: CommonPaymentService
 
   def sendETDataExtensionRow(subscribeResult: SubscribeResult, subscriptionData: SubscriptionData): Future[Unit] = {
-    val subscription = zuoraService.getSubscription(Subscription.Name(subscribeResult.subscriptionName))
+      val subscription = subscriptionService.unsafeGetPaid(Subscription.Name(subscribeResult.subscriptionName))(Digipack)
+      val paymentMethod = paymentService.getPaymentMethod(Subscription.AccountId(subscribeResult.accountId)).map(
+        _.getOrElse(throw new Exception(s"Subscription with no payment method found, ${subscribeResult.subscriptionId}"))
+      )
 
     val accAndPaymentMethod = for {
```

Comment:
`accAndPaymentMethod` is probably misnamed now, and looks like this stuff should be merged together:

https://github.com/guardian/subscriptions-frontend/blob/3f50bef63d/app/services/ExactTargetService.scala#L33-L40